Tasks/PublishCodeCoverageResultsV2/task.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "2A7EBC54-C13E-490E-81A5-D7561AB7CD97",
+    "name": "PublishCodeCoverageResults",
+    "friendlyName": "Publish code coverage results",
+    "description": "Publish any of the  code coverage results supported by report generator from a build",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/test/publish-code-coverage-results",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=626485)",
+    "category": "Test",
+    "visibility": [
+        "Build"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 2,
+        "Minor": 158,
+        "Patch": 6
+    },
+    "demands": [],
+    "minimumAgentVersion": "2.102.0",
+    "instanceNameFormat": "Publish code coverage results",
+    "inputs": [
+        {
+            "name": "summaryFileLocation",
+            "type": "multiLine",
+            "label": "Summary file",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Paths to summary files containing code coverage statistics, such as line, method, and class coverage. The value may contain minimatch patterns. For example: `$(System.DefaultWorkingDirectory)/MyApp/**/site/cobertura/coverage.xml`. Supports multiple lines of minimatch patterns. [More information](https://aka.ms/minimatchexamples)"
```

Comment:
We are calling out minimatch multiple times. Remove "The value may …" the second reference is good enough.